src/apps/expenses/components/CreateExpenseForm.js

```diff
@@ -463,7 +463,7 @@ class CreateExpenseForm extends React.Component {
                   type="email"
                   name="paypalEmail"
                   key={`paypalEmail-${get(LoggedInUser, 'id')}`}
-                  defaultValue={this.state.expense.paypalEmail}
```

Comment:
Why this change?

Reply:
Initially, `paypalEmail` is `null` before the prop updates. `defaultValue` binds to the initial value during the first render so it doesn't change if the user is actually logged in and has a default email.
This change fixes that.